dot/state/block.go

```diff
@@ -773,10 +773,14 @@ func (bs *BlockState) IsDescendantOf(ancestor, descendant common.Hash) (bool, er
 			return false, fmt.Errorf("getting header: %w", err2)
 		}
 
-		for current := descendantHeader; descendantHeader.Number < ancestorHeader.Number; {
+		for current := descendantHeader; current.Number < ancestorHeader.Number; {
 			if current.ParentHash == ancestor {
 				return true, nil
 			}
+			current, err2 = bs.GetHeader(current.ParentHash)
```

Comment:
```suggestion
			current, err = bs.GetHeader(current.ParentHash)
```

Reply:
discussed over meeting, decided to ignore it